core_api/tests/conftest.py

```diff
@@ -9,11 +12,22 @@
 from jose import jwt
 from langchain_community.embeddings import SentenceTransformerEmbeddings
 from langchain_community.llms.fake import FakeListLLM
+from langchain_core.embeddings.fake import FakeEmbeddings
+from langchain_core.documents.base import Document
+from langchain_elasticsearch import ElasticsearchStore
+from langchain_elasticsearch.retrievers import ElasticsearchRetriever
+from langchain_core.runnables import ConfigurableField
 
 from core_api.src.app import app as application
 from redbox.model_db import MODEL_PATH
 from redbox.models import Chunk, File, Settings
+from redbox.models.file import Metadata
 from redbox.storage import ElasticsearchStorageHandler
+from core_api.src.retriever import get_all_chunks_query, ParameterisedElasticsearchRetriever
+from core_api.src.retriever.parameterised import parameterised_document_mapper
+
+ALL_CHUNKS_RETRIEVER_RESOURCES = json.load(open("core_api/tests/resources/retriever/all_chunks.json"))
```

Comment:
pls can we just encode these as python? i dont think we are gaining anything by having them in json and it is conceivable (if unlikely) that the IDE might be able to help with any errors in the params.